virttest/qemu_vm.py

```diff
@@ -2159,7 +2159,7 @@ def sort_key(dev):
             ovmf_vars_path = "%s.fd" % path
             devs = []
             pflash0, pflash1 = ('ovmf_code', 'ovmf_vars')
-            if Flags.BLOCKDEV in self._caps:
+            if Flags.BLOCKDEV in vm._caps:
```

Comment:
Suggest invoking `vm.check_capability(Flags.BLOCKDEV)` instead of reading the private attribute directly.

Reply:
Sure, we still need it in qcontainer, though...